docs/tutorial/binding.md

```diff
@@ -181,6 +181,17 @@ Both let-bindings and function parameters support pattern matching. This is a
 very powerful mechanism to extract elements from a complex structure and also
 restrict what values may be used as input.
 
+The bare literals are supported:
+
+```arrai
+@> let 42 = 42; 1
+@> let "hello" = "hello"; 1
+@> let 3 = 1 + 2; 5
+@> let (1 + 2) = 3; 5
```

Comment:
This isn't strictly a bare literal. I suggest an additional block with a couple of examples, with lead-in:
> Complex expressions are supported and will also match their own value. However, they must be enclosed in parentheses, `(...)`: